### ISSUE TITLE
mbmore CascadeEnrich archetype fails tests

### ISSUE THREAD

**Issue by danielwojtaszek** [CNERG/mbmore#31]
I've installed Mbmore from https://github.com/CNERG/mbmore but some unit tests for CascadeEnrich failed. I've attached the mbmore_unit_tests output.  I have CYCLUS 1.5.5 installed on Ubuntu 16.04.

The mbmore commit info is:

commit da1e0ac7b24de4a1c3f6064181cae1d774913f29
Merge: ab6e40d 5dd271c
Author: Paul Wilson <paul.wilson@wisc.edu>
Date:   Tue Sep 15 06:27:21 2020 -0500
[test_results.txt](https://github.com/CNERG/mbmore/files/7034104/test_results.txt)

**Comment by bam241** [comment]
The last time I check CascadeEnrich was highly experimental...
the main version is Cleary out of date.

my "last" updated branch was `CascadeEnrich_PR_branch`

The last person working on this was @stompsjo  he probably owns the most up-to-date version of it....

**Comment by stompsjo** [comment]
Correct, [stompsjo:dU_SWU_correction](https://github.com/stompsjo/mbmore/tree/dU_SWU_correction) is my latest development branch. I am not familiar with what currently exists on the main branch. The most recent [PR](https://github.com/CNERG/mbmore/pull/29) from 14 months ago also noted the failure of these tests before and after the changes of that pull request.

AFAIK `stompsjo:dU_SWU_correction` passes all unit tests, but several are changed from what is currently on `CNERG/mbmore:main`.
